Source/JavaScriptCore/b3/air/AirOpcode.opcodes

```diff
@@ -309,6 +309,30 @@ arm64: MultiplySignExtend32 U:G:32, U:G:32, D:G:64
 arm64: MultiplyZeroExtend32 U:G:32, U:G:32, D:G:64
     Tmp, Tmp, Tmp
 
+arm64: MulHigh32 U:G:32, U:G:32, ZD:G:32
+    Tmp, Tmp, Tmp
+
+arm64: UMulHigh32 U:G:32, U:G:32, ZD:G:32
+    Tmp, Tmp, Tmp
+
+arm64: MulHigh64 U:G:64, U:G:64, D:G:64
+    Tmp, Tmp, Tmp
+
+arm64: UMulHigh64 U:G:64, U:G:64, D:G:64
+    Tmp, Tmp, Tmp
+
+x86: X86MulHigh32 U:G:32, UD:G:32, D:G:32
+    Tmp, Tmp*, Tmp*
```

Comment:
What's star in `Tmp*` again? It seems `Tmp` means the allocator can pick any register. But we must pick a specific, pre-defined physical register (e.g., eax, edx) for `Tmp*`.

Reply:
This means that these registers have special constraint, so AirInstInlines.h's helper function will be invoked to validate them.